src/dotnet-user-secrets/Program.cs

```diff
@@ -71,6 +71,12 @@ internal int RunInternal(params string[] args)
 
             var reporter = CreateReporter(options.IsVerbose);
 
+            if (options.Command is InitCommand initCmd)
+            {
+                initCmd.Execute(new CommandContext(null, reporter, _console), _workingDirectory);
+                return 0;
```

Comment:
In retrospec, Command.Execute should probably have returned `int` so the command can set an exit code. I'd be happy to also take a PR that made this change.

Reply:
Okay - I looked at that but ran into the fact the interface had the expectation. At first glance though, I don't know how I'd safely modify the interface to allow a status code return without causing a breaking change (but that's a separate conversation).

Reply:
You have my blessing to make a breaking change on `ICommand`. For projects like this, the `ICommand` type is an internal implementation detail of these console tools. No one should be compiling an app against `dotnet-user-secrets.dll`.

Reply:
The `ICommand` in this instance is from CommandLineUtils, so not part of this tool - do you still want that change?